Automattic/jetpack
# WAF: Add loading and running from Jetpack plugin

#### Changes proposed in this Pull Request:

* Moves most support and loading functionality out of actions.php
* Functionality to run the WAF is moved to the plugin running the package
* Other business logic and hooks have been moved to a package class
* Minor standardization updates to the rest api class

#### Jetpack product discussion



Asana [#1202377528124263](https://app.asana.com/0/1201069996155224/1202377528124263)

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* After applying changes, ensure that bad requests still get blocked
* Test both modes
* Turn on and off settings to ensure it works
* [See this post for more information](https://wp.me/peb6dq-3W)
